libr/asm/asm.c

```diff
@@ -84,19 +86,20 @@ static inline int r_asm_pseudo_intN(RAsm *a, RAsmOp *op, char *input, int n) {
 		return 0;
 	}
 	// XXX honor endian here
```

Comment:
Why `// XXX`? Seems already honored.

Reply:
nope. its saving it in little endian always

Reply:
but somenthing to fix in another pr, not here